index.js

```diff
@@ -0,0 +1,45 @@
+const express = require('express')
+const fs = require('fs')
+const app = express()
+const port = 3000
+
+const handleOfficeHours = (req, res, next) => {
+    const date = new Date()
+    const hour = date.getHours()
+    const day = date.getDay()
+
+    if(day != "Saturday" && day != "Sunday"){
```

Comment:
The `day` variable will not be a string containing the name of the week day. It returns an integer representing the day of the week ranging from 0 to 6. Learn more about it in its [W3Schools documentation](https://www.w3schools.com/jsref/jsref_getday.asp).

It also would be better to create a descriptive variables then use them:

```js
const now = new Date();
const hour = date.getHours();
const today = date.getDay();

const isWorkday = today !== 0 && today !== 6;
const isWithinWorkingHours = hour >= 9 && hour < 17;

if (isWorkday && isWithinWorkingHours ) {
  next();
  return;
}

const data = fs.readFileSync("./out-office.html");
const html = data.toString();

res.status(301).set("Content-Type", "text/html").send(html);
```